### ISSUE TITLE
Alias ~ to package name

### ISSUE THREAD

**Issue by matthewp** [stealjs/steal#719]
We'd like to be able to write:

```js
import foo from "~/foo";
```

Rather than

```js
import foo from "longPackageName/foo";
```

This should be relatively easy, we call `parseFromPackage` in the npm plugin [here](https://github.com/stealjs/system-npm/blob/d4b0129f1242ad520cc2314d580f736f572c4dfd/npm-extension.js#L88) which in turn calls `parse` [here](https://github.com/stealjs/system-npm/blob/d4b0129f1242ad520cc2314d580f736f572c4dfd/npm-utils.js#L146).  parse should be changed to see the ~ and know the package name from that.

**Comment by matthewp** [comment]
Going to add this to the 1.0.0 milestone as it should be fairly easy to squeeze in, will remove it if it turns out not to be. Anyone who wants it is welcome to take it.

**Comment by pYr0x** [comment]
@matthewp can you also pls make a steal 1.0 pre release?

**Comment by matthewp** [comment]
Just did.

**Comment by pYr0x** [comment]
worked on major branch...

**Comment by pYr0x** [comment]
Reopen it because it needs to be documented

**Comment by matthewp** [comment]
@pYr0x does this require that you use `directories.lib`, I wouldn't think so, correct?

**Comment by matthewp** [comment]
Closed by #748
